arxiv/base/urls/links.py

```diff
@@ -110,7 +113,7 @@ def _without_group_names(pattern: Pattern) -> str:
 )
 """Regex to match an IP address."""
 
-TLDS = "|".join( list(set(bleach.linkifier.TLDS + ['xyz'])) )
+TLDS = "[a-z][a-z]+"     # TLDs are out of control. Just match on two or more alphabetic characters.
```

Comment:
`[a-z]{2,24}` if you'd like to keep some defensive posture in place, which fits [today's entries](https://data.iana.org/TLD/tlds-alpha-by-domain.txt)

Reply:
Just noticing, that list contains numbers, uppercase, and hyphens: 
eg: XN--11B4C3D

Reply:
Right... Unsure if arXiv wants to support the international/punycode TLDs.

I found this illustration useful:
https://stackoverflow.com/a/73155251

Reply:
The matching is case-insensitive (elsewhere).

I don't see the need to highlight the other URLs yet.